warp/src/main/java/com/schibsted/nmp/warp/components/WarpCheckbox.kt

```diff
@@ -64,55 +85,165 @@ fun WarpCheckbox(
             warpCheckboxColors = checkboxStyleDisabled()
         }
 
+        if (isError) {
+            warpCheckboxColors = checkboxStyleNegative()
+        }
+
         WarpCheckboxView(
+            modifier = modifier,
             text = text,
+            subText = subText,
             onCheckedChange = onCheckedChange,
             warpCheckboxColors = warpCheckboxColors,
             checked = checked,
-            enabled = ((style != WarpCheckboxStyle.Disabled) && enabled),
-            modifier = modifier
+            enabled = ((style != WarpCheckboxStyle.Disabled) && enabled)
         )
 
     }
 }
 
-// Custom checkbox view as the Material 3 checkbox doesn't support setting the border color separately
+/**
+ * A group of checkboxes in the warp design system.
+ * For more info, look [here](https://warp-ds.github.io/tech-docs/components/checkbox/)
+ *
+ * @param modifier The modifier to be applied to the checkbox group.
+ * @param orientation The orientation of the checkbox group. Default is Orientation.Vertical
+ * @param title The optional title of the checkbox group.
+ * @param options The list of options to be displayed in the checkbox group.
+ * @param selectedOption The list of selected options. Default is null.
+ * @param helpText The optional help text of the checkbox group.
+ * @param enabled Set to false to disable the checkbox group. Default value is true.
+ * @param isError Set to true if the checkbox group is in an invalid state. Default value is false.
+ * @param onOptionSelected A lambda that will be invoked when an option is selected. Returns a mutable list of selected options.
+ */
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun WarpCheckboxGroup(
```

Comment:
This file has grown a bit large, don't you think the group could move to its own file? It doesn't really fit to be in "WarpCheckBox"...